src/core/product-transformer.ts

```diff
@@ -71,6 +71,35 @@ namespace Utils {
       return Utils.remap(transformed, effectiveStructure, baseProduct);
     };
   }
+
+  export const transformData = (data: any) => {
+    return ({
+      ...data,
+      price: transformPrice(data.price),
+    });
+  };
+
+  // note that this assumes data won't have thousands separators
+  export const transformPrice = (price: string): string => {
+    if (!price) {
+      return '';
+    }
+    const PRICE_REGEX = /^(.*)([\.,])(.*)$/;
+    const match = PRICE_REGEX.exec(price);
+    if (match) {
+      return match[1] + match[2] + simplePad(match[3].substr(0,2));
+    } else {
+      return price;
+    }
+  };
+
+  export const simplePad = (input: string) => {
```

Comment:
Cute name :) But can we try something like "decimalPad"?

Reply:
But nice Regex btw :+1:

Reply:
There isn't a reliable way to use `Number.prototype.toFixed()`, is there?

Reply:
No, we get strings from the api response.